src/EStimLibrary/Extensions/SpatialModel/StringHierarchy/StringHierarchyAreaFactory.cs

```diff
@@ -20,7 +20,7 @@ public StringHierarchyAreaFactory(StringHierarchyRegion baseModelRegion)
 
         this.HelpMsg = $"A StringHierarchyArea can be built from one of the " +
             $"following path specs, selecting one option from any list in [] " +
-            $"and excluding the []:\n{baseModelRegion.ToString()}";
+            $"and excluding the []:\n{baseModelRegion?.ToString() ?? "null"}";
```

Comment:
In which cases were you finding baseModelRegion was null? It should always have a value after object initialization, so we don't really want to allow the possibility that it's null.